.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+# Description
+
+*Please provide a detailed description. Be as descriptive as possible - include information about what is being changed,
+why it's being changed, and any links to relevant issues. If this is closing an existing issue use one of the [issue linking keywords](https://docs.github.com/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to link the issue to this PR and have it automatically close when completed.*
+
+In addition, go through the checklist below and check each item as you validate it is either handled or not applicable to this change.
+
+# Code Changes
+
+- [ ] [Unit tests](https://github.com/microsoft/SqlScriptDOM/tree/main/Test) are added, if possible
+- [ ] New or updated code follows the guidelines [here](https://github.com/microsoft/SqlScriptDOM/blob/main/CONTRIBUTING.md#helpful-notes-for-sqldom-extensions)
+
```

Comment:
- [ ] Existing [tests are passing](https://github.com/microsoft/SqlScriptDOM/blob/main/CONTRIBUTING.md#running-the-tests)